roles/kubernetes-apps/heapster/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Heapster | Lay Down Heaspter Templates
+  template: src={{item.file}} dest={{kube_config_dir}}/{{item.file}}
+  with_items:
+    - {file: grafana-deployment.yaml, type: deployment, name: monitoring-grafana}
+    - {file: grafana-service.yaml, type: svc, name: monitoring-grafana}
```

Comment:
nit: change filenames to be grafana-svc.yml and grafana-deploy.yml (according to kubectl official short form names)